cupy_backends/cuda/libs/cusparse.pxd

```diff
@@ -103,9 +103,6 @@ cpdef enum:
     CUSPARSE_SOLVE_POLICY_NO_LEVEL = 0
     CUSPARSE_SOLVE_POLICY_USE_LEVEL = 1
 
-    CUSPARSE_ALG_NAIVE = 0
```

Comment:
Why are only these two enums removed from here and moved under the IF condition?

Reply:
@bmedishe Could you review this one with 6.0_internal_testing branch. Looks like these enum values shouldn't be changed.

Reply:
@lcskrishna  these two enums do not have hip mapping